libs/core/src/scss/base/_variables.scss

```diff
@@ -188,3 +188,7 @@ $transition-easings: (
   ),
 );
 $loading-overlay-backdrop-opacity: 0.8;
+$form-field-input-font-family: var(--kirby-font-family);
+$form-field-input-line-height: 1.5;
+$form-field-input-padding: 1em;
+$form-field-label-height: 24px;
```

Comment:
These Sass variables should stay in `libs/designsystem/src/lib/components/form-field/_form-field-inputs.shared.scss`

Reply:
If they stay there. Then how can I import them without all the styles getting applied as well?

Reply:
See https://github.com/kirbydesign/designsystem/pull/2390#discussion_r913591504